src/main/java/Gops/Parser.java

```diff
@@ -0,0 +1,119 @@
+package Gops;
+
+import java.io.File;
+public class Parser {
+    /**
+     * Parses the user input and does the appropriate operation on the taskList and updates the hard disk
+     * @param taskList list of tasks
+     * @param txtFile file containing tasks
+     */
+    public String parse(String userReply, TaskList taskList, File txtFile) {
```

Comment:
This method is quite long, [issue 303](https://github.com/nus-cs2103-AY2324S2/forum/issues/303) on the forum may be  of interest.